src/main/kotlin/org/jetbrains/research/ml/ast/transformations/PyElementGeneratorExt.kt

```diff
@@ -0,0 +1,35 @@
+package org.jetbrains.research.ml.ast.transformations
+
+import com.jetbrains.python.psi.LanguageLevel
+import com.jetbrains.python.psi.PyBoolLiteralExpression
+import com.jetbrains.python.psi.PyElementGenerator
+import com.jetbrains.python.psi.PyExpression
+import com.jetbrains.python.psi.PyPrefixExpression
+
+// TODO: merge this file with PyUtils?
+private val defaultLanguageLevel = LanguageLevel.getDefault()
+
+fun PyElementGenerator.createBoolLiteralExpression(value: Boolean): PyBoolLiteralExpression {
+    return createExpressionFromText(defaultLanguageLevel, if (value) "True" else "False") as PyBoolLiteralExpression
```

Comment:
I don't really know which is better, but I'll just suggest. We can also use the following code:
```
value.toString().capitalize()
```

Reply:
I think we should keep it as is, since `value.toString().capitalize()` unnecessarily ties together keywords from Python and Kotlin.